dotnet/src/SemanticKernel.Core/Functions/ContextualSelection/ContextualFunctionProvider.cs

```diff
@@ -55,13 +56,15 @@ public sealed class ContextualFunctionProvider : AIContextProvider
     /// <param name="vectorDimensions">The number of dimensions to use for the memory embeddings.</param>
     /// <param name="functions">The functions to vectorize and store for searching related functions.</param>
     /// <param name="maxNumberOfFunctions">The maximum number of relevant functions to retrieve from the vector store.</param>
+    /// <param name="loggerFactory">The logger factory to use for logging. If not provided, no logging will be performed.</param>
     /// <param name="options">The provider options.</param>
     /// <param name="collectionName">The collection name to use for storing and retrieving functions.</param>
     public ContextualFunctionProvider(
         VectorStore vectorStore,
         int vectorDimensions,
         IEnumerable<AIFunction> functions,
         int maxNumberOfFunctions,
+        ILoggerFactory? loggerFactory = default,
```

Comment:
nit.
Normally I see `null`, curious why is different here.

```suggestion
        ILoggerFactory? loggerFactory = null,
```